src/dockerflow/flask/app.py

```diff
@@ -375,42 +352,3 @@ def my_version(root):
 
         """
         self._version_callback = func
-
-    def check(self, func=None, name=None):
```

Comment:
@leplatrem speaking of breaking changes, this would be a breaking change as well right?

Maybe we should preserve `Dockerflow.check`, but include a deprecation warning and call `checks.register` inside for backwards compatibility? Same for Sanic.

Reply:
Yeah, I would vote for this route if the amount of efforts is not tremendous. This package may be used in projects with low maintenance resources, and avoid breaking changes can be appreciated